Src/FluentAssertions/NumericAssertionsExtensions.cs

```diff
@@ -43,11 +45,7 @@ public static class NumericAssertionsExtensions
                 maxValue = sbyte.MaxValue;
             }
 
-            Execute.Assertion
-                .ForCondition(minValue <= actualValue && actualValue <= maxValue)
-                .BecauseOf(because, becauseArgs)
-                .FailWith("Expected {context:value} to be within {0} from {1}{reason}, but found {2}.",
-                    delta, nearbyValue, actualValue);
+            BeCloseToFailIfValueOutsideBounds(minValue <= actualValue && actualValue <= maxValue, nearbyValue, delta, actualValue, because, becauseArgs);
```

Comment:
🙃 `BeCloseToFailIfValueOutsideBounds` is a bit of a weird name. I'm not sure how to read it.

Reply:
Name simplified.